modules/claims_api/spec/requests/v2/veterans/claims_request_spec.rb

```diff
@@ -316,7 +316,8 @@
                     {
                       base_end_prdct_type_cd: '400',
                       benefit_claim_id: '111111111',
-                      claim_status: 'Preparation for notification'
+                      claim_status: 'PEND',
+                      phase_type: 'Gathering of Evidence'
```

Comment:
I believe BGS only provides PEND, COMPLETE and CAN for their status, we should pull from phase type if outside of those. Phase Type pulled from cassette, so adjusted to Gathering of Evidence (also impacted Spec result